test/unit/utils/BaseFactory.mocha.ts

```diff
@@ -0,0 +1,69 @@
+/// <reference path='../test.mocha.ts' />
+
+class BaseFactoryStub extends n3Charts.Utils.BaseFactory {
```

Comment:
the stub in the FactoryManager is already called FactoryStub, so I named it like this. But you are absolutely right, it needs a more descriptive name. what about `ChildFactoryStub`